src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
```

Comment:
🙂 I can see the same issue over here your body is empty there is nothing in your body, and this could cause a lighthouse error. I think you can put at least 1 HTML element to solve this issue.